doc/api/net.md

```diff
@@ -600,6 +600,10 @@ the error passed to the [`'error'`][] listener.
 The last parameter `connectListener`, if supplied, will be added as a listener
 for the [`'connect'`][] event **once**.
 
+This function should not be used for reconnecting a socket. Calling
+[`socket.connect()`][] a second time leads to undefined behavior.
+[`net.connect()`][] implicitly calls [`socket.connect()`][].
```

Comment:
Why is this line needed?

Reply:
Which line are you referring to specifically?

Reply:
You mean `net.connect`? Because if you use `net.connect` you haven't technically called `socket.connect`, hence the user might believe it should be fine to called it the "first" time.